lib/danger.rb

```diff
@@ -1,63 +1,14 @@
 require "danger/version"
 require "danger/dangerfile"
 require "danger/environment_manager"
+require "danger/runner"
+require "danger/init"
 
 require 'claide'
 require 'colored'
+require 'pathname'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.